internal/xsql/conn.go

```diff
@@ -139,7 +138,7 @@ func (c *conn) PrepareContext(ctx context.Context, query string) (_ driver.Stmt,
 
 func (c *conn) execContext(ctx context.Context, query string, args []driver.NamedValue) (_ driver.Result, err error) {
 	m := queryModeFromContext(ctx, c.defaultQueryMode)
-	onDone := trace.DatabaseSQLOnConnExec(c.trace, &ctx, query, m.String(), retry.IsIdempotent(ctx))
+	onDone := trace.DatabaseSQLOnConnExec(c.trace, &ctx, query, m.String(), xerrors.IsIdempotent(ctx))
```

Comment:
What about move IsIdempotent, WithIdempotent from xerrors to xcontext?
```
xcontext.IsIdempotent(ctx)
```

Reply:
ack